onboarding/main.go

```diff
@@ -101,7 +142,8 @@ func newClient() (*kubernetes.Clientset, error) {
 	clientset, err := kubernetes.NewForConfig(config)
 
 	if err != nil {
-		klog.Error(err, "failed to get clientset")
+		klog.Errorf("failed to create clientset: %v", err)
+		os.Exit(1)
```

Comment:
better to exit only from main and `return nil, err` here, wdyt?

Reply:
Make sense, That is how we followed, Will do the change

Reply:
Done.